### ISSUE TITLE
Could not build Objective-C module?

### ISSUE THREAD

**Issue by schan93** [stephencelis/SQLite.swift#120]
Hi, I am having an issue with SQLite. Ever since I have updated my version of XCode to version 6.3, I have been getting an issue saying that it cannot find the module SQLite3 when I try to import it into my files using `import SQLite`. As a result, I resorted to deleting the entire SQLite folder from my project and re-adding it to my project. However, now I am getting issues saying `Header '/usr/include/sqlite3.h' not found` and `Could not build Objective-C module 'sqlite3`. It was all working for me previous to this but now I have no idea and every time i re-add the folder to my project, I get this error or an error saying `Coudl not build Objective-C module SQLite `. Any help would be appreciated. Thanks!

**Comment by stephencelis** [comment]
Hm, that might be related to this commit:

https://github.com/stephencelis/SQLite.swift/commit/2bf90d49cc982589b6d7ed9585f3f8f41ccf3263

Can you try checking out the commit just prior and see if that solves your problem?

https://github.com/stephencelis/SQLite.swift/commit/213920b2f65d7ae92bb2c93cfa02a2e6e19f8ee9

(Here's a download link if you're not using git: https://github.com/stephencelis/SQLite.swift/archive/213920b2f65d7ae92bb2c93cfa02a2e6e19f8ee9.zip)

**Comment by schan93** [comment]
Ah that seems to work better but now I am getting an error saying `Cannot load underlying module for 'XCTest'`. Any ideas on how to fix this?

**Comment by DannyGlover** [comment]
I'm also getting the same issue, after updating to commit: https://github.com/stephencelis/SQLite.swift/commit/2bf90d49cc982589b6d7ed9585f3f8f41ccf3263

**Comment by stephencelis** [comment]
Reverted that commit, so `master` should hopefully work fine now!

@schan93 As far as `Cannot load underlying module for 'XCTest'` goes, it means you're somehow linking against the test target. Please re-read the installation instructions and avoid any cases of **SQLite Tests**, etc.

**Comment by WhitneyMFoster** [comment]
"import Foundation" fixed it for me

**Comment by markie2902** [comment]
this bug has been fixed right? I'm getting the "Could not build Objective-C module 'SQLite' "

**Comment by clehew** [comment]
I'm getting this as well, along with "Header 'SQLite-Swift.h' not found."

**Comment by jberkel** [comment]
@clehew how do you integrate SQLite.swift into your project?

**Comment by clehew** [comment]
@jberkel  CocoaPods.  XCode 8.2, Cocoapods v1.1.1, pod 'SQLite.swift', '~> 0.11.1'

**Comment by jberkel** [comment]
@clehew and Xcode installed in `/Applications/Xcode.app`?

**Comment by clehew** [comment]
@jberkel  yes

**Comment by clehew** [comment]
@jberkel  I moved it, still didn't work, then I uninstalled/reinstalled, still not working, same error

**Comment by jberkel** [comment]
@clehew. hmm, weird. can you try if a minimal project via `pod try SQLite.swift` compiles ok?

**Comment by ehsan-black** [comment]
@clehew Hi, I've had the same issue today(Xcode 8.2, pod 'SQLite.swift', '~> 0.11.1' ).
it's been fixed with moving Xcode to /Applications/Xcode.app and running this line in terminal:
sudo xcode-select --switch /Applications/Xcode.app

**Comment by jberkel** [comment]
@ehsan-black thanks for the hint, will add it to the instructions

**Comment by clehew** [comment]
@jberkel  Sorry for the slow response, I have been away on vacation, but I'm still fighting this strange error.  I know now that is probably not Sqlite.swift.  I have managed to see the error change to SnapKit-swift.h on a few occasions and I was able to create a bare bones project that built fine.  this is the set of errors I'm getting if you have any ideas.

I'm at a loss at this point, I've event updated to the latest cocoapods beta because some people claimed that helped for another library. 

![screen shot 2017-01-02 at 5 16 46 pm](https://cloud.githubusercontent.com/assets/14948084/21597350/455f4fa2-d10f-11e6-9b23-313f2e3555a6.png)
